lib/kubeclient/common.rb

```diff
@@ -300,9 +305,44 @@ def create_rest_client(path = nil)
       RestClient::Resource.new(@api_endpoint.merge(path).to_s, options)
     end
 
+    # *DEPRECATED:* Use +http_client+ instead.
```

Comment:
Formally, `rest_client` is an internal method users should not rely on anyway because it's not documented in README.
it wasn't made `private` and I've seen some people use it but it's OK to break it if you wish :shrug: